src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -132,10 +163,10 @@ public ZonedDateTime time() throws IOException {
     @Override
     public long amount() throws IOException {
         final String amnt = new UncheckedText(
-            new IoCheckedScalar<>(
-                new ItemAt<>(
-                    2, new SplitText(this.transaction, ";")
-                )
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
```

Comment:
@paulodamaso same here, we can avoid the lambda here and re-wrapping `String` by changing `transaction` to `IoCheckedScalar`.